lib/Core/formatNumberForLocale.js

```diff
@@ -32,10 +35,14 @@ function formatNumberForLocale(number, options) {
     options.useGrouping = defaultValue(options.useGrouping, false);
     options.maximumFractionDigits = defaultValue(options.maximumFractionDigits, 20);
     if (toLocaleStringSupportsOptions()) {
+        // The order of options may lead to equivalent keys being cached multiple times, but this is still better than creating a new Intl.NumberFormat every time.
```

Comment:
Yes... but I'm not sure it's worth it. The real gain here is saving recreating the formatter 1000s of times (once per row of the csv file). In contrast the minor performance hit from caching the same formatter twice (or even several times), instead of once, doesn't seem too bad, vs. the complexity of adding the extra code. What do you think?